### PR TITLE
liuliu/bump-cypress-to-13.1.0

### PR DESCRIPTION
**Pull request by liuliu-dev**

### CODE REVIEW

File: cypress/e2e/dolthub/publicPaths/render/database/tags.spec.ts [hunk 1]
```diff
@@ -165,20 +99,20 @@ export const testCommitSection = (commitLen: number): Expectation =>
...
+const tests = (n: number) => {
+  const num = n + 2;
+  return [
+    newExpectation(
+      "should not find empty database",
+      "[data-cy=repo-data-table-empty]",
+      notExist,
+    ),
+    testCommitSection(num),
+    // testReleasesSection(21),
```
Comment (tbantle22) [inline cypress/e2e/dolthub/publicPaths/render/database/tags.spec.ts]: Why is this commented out?

File: cypress/e2e/utils/index.ts [hunk 2]
```diff
@@ -48,95 +48,65 @@ export const deviceDimensions: Record<
...
-
-        if (t.scrollTo) {
-          handleScrollTo(t.scrollTo);
-        }
-
-        if (t.redirect) {
-          // Sign out after signing in for redirect and running tests
-          cy.signout(isMobile);
-        }
+    if (t.skip) return;
```
Comment (tbantle22) [inline cypress/e2e/utils/index.ts:57]: Should log description before this line

File: cypress/e2e/utils/index.ts [hunk 3]
```diff
@@ -48,95 +48,65 @@ export const deviceDimensions: Record<
...
-      describe.skip(d.description, deviceDimensions[d.device], () => {
-        runTests({ ...d, currentPage });
+      xit(d.description, deviceDimensions[d.device], () => {
+        runTests(d);
       });
     } else {
-      describe(d.description, deviceDimensions[d.device], () => {
+      it(d.description, deviceDimensions[d.device], () => {
+        it(`visiting ${currentPage}`, () => {
+          cy.visitPage(currentPage, loggedIn);
```
Comment (tbantle22) [inline cypress/e2e/utils/index.ts:97]: Can this be moved to a `beforeEach` like we do in hosted-cypress?

File: cypress/e2e/utils/index.ts [hunk 4]
```diff
@@ -48,95 +48,65 @@ export const deviceDimensions: Record<
...
+        });
+
         // TODO: This error comes from fetching github stars for the navbar. We should fix eventually
         if (ignoreUncaughtErrors) {
           it("should ignore Gatsby server error", () => {
             cy.ignoreUncaughtErrors(gatsbyServerBuildErrors);
           });
         }
-        runTests({ ...d, currentPage });
+        it(d.description, () => {
```
Comment (tbantle22) [inline cypress/e2e/utils/index.ts:106]: Should this be...
```suggestion
       it(d.description, deviceDimensions[d.device], () => {
```

**Review by tbantle22 — COMMENTED** [review]
Overall looks good. Glad you got this to work! A few comments

File: cypress/e2e/dolthub/publicPaths/render/database/tags.spec.ts (hunk 1 above)
Comment (liuliu-dev) [reply cypress/e2e/dolthub/publicPaths/render/database/tags.spec.ts]: it was for debugging, uncommented it back now

**Pull request merged**